### ISSUE TITLE
Image trasfer ain't works

### ISSUE THREAD

**Issue by iuvi7** [quiet/quiet-js#50]
Hello!

I found on site of quiet-js ([here](https://quiet.github.io/quiet-js/)) there's and option to send Images but Reciver doesn't work for me either from notebook or my android :(

Anyone have same problem? 

**_And if_** there any other **_software to send files over sound_** between PC's and\or Android devices? 

Thanks

**Comment by The-Best-Codes** [comment]
@iuvi7 I have the same issue.

**Comment by brian-armstrong** [comment]
@The-Best-Codes @iuvi7 

Hello,

Just to confirm, as instructed on the demo, do you have the sender and receiver connected via an audio cable?

**Comment by The-Best-Codes** [comment]
@brian-armstrong I was not aware of this requirement, thanks.
Why does the page freeze if I don't have anything connected via a cable?

**Comment by iuvi7** [comment]
> [@The-Best-Codes](https://github.com/The-Best-Codes) [@iuvi7](https://github.com/iuvi7)
> 
> Hello,
> 
> Just to confirm, as instructed on the demo, do you have the sender and receiver connected via an audio cable?

Yes, i connected my second laptop with another one over 3.5 jack -> messages flying well, but files are don't (same with connecting android to laptop by jack)

It freezes if i use online version, but if offline mode (by Largon local server) more or less no-freezes
